server/src/pages/shot/view.js

```diff
@@ -420,15 +420,15 @@ class ExpireWidget extends React.Component {
     let day = hour * 24;
     return (
       <span>
-        keep for <select ref="expireTime">
-          <option value="cancel">Select time:</option>
-          <option value="0">Indefinitely</option>
-          <option value={ 10 * minute }>10 Minutes</option>
-          <option value={ hour }>1 Hour</option>
-          <option value={ day }>1 Day</option>
-          <option value={ 7 * day }>1 Week</option>
-          <option value={ 14 * day }>2 Weeks</option>
-          <option value={ 31 * day }>1 Month</option>
+        keep for <select ref="expireTime"> <!-- todo l10n -->
```

Comment:
Speaking of declensions, does it make sense to have "keep for: " separate from the time period?